.github/workflows/publish_docker.yaml

```diff
@@ -39,10 +39,45 @@ jobs:
           else
               echo ::set-output name=match::false
           fi
+      - name: Check RC Tag
+        id: check-rc-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+\.[0-9]+\.[0-9]+rc[0-9]+$ ]]; then
+              echo ::set-output name=match::true
+          else
+              echo ::set-output name=match::false
+          fi
+      - name: Check alpha Tag
+        id: check-alpha-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+\.[0-9]+\.[0-9]+a[0-9]+$ ]]; then
+              echo ::set-output name=match::true
+          else
+              echo ::set-output name=match::false
+          fi
+      - name: Check beta Tag
+        id: check-beta-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+\.[0-9]+\.[0-9]+b[0-9]+$ ]]; then
+              echo ::set-output name=match::true
+          else
+              echo ::set-output name=match::false
+          fi
+
       - name: Push Fides Prod Tags
         if: steps.check-tag.outputs.match == 'true'
         run: nox -s "push(prod)"
 
+      # if an RC tag, then we run the rc job to publish images with an ":rc" tag
+      - name: Push Fides RC Tags
+        if: steps.check-rc-tag.outputs.match == 'true'
+        run: nox -s "push(rc)"
+
+      # if an alpha or beta tag, then we run the prerelease job to publish images with an ":prerelease" tag
+      - name: Push Fides prerelease Tags
+        if: steps.check-alpha-tag.outputs.match == 'true' || steps.check-beta-tag.outputs.match == 'true'
+        run: nox -s "push(rc)"
```

Comment:
should this be "rc"? Not "prerelease"?

Reply:
good eyes! thank you 🙏 ... silly c+p error 🤦

Reply:
fixed in https://github.com/ethyca/fides/pull/3474/commits/602a7312c306053aafba9cf41da3b8f278993f10